pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -69,11 +221,54 @@ func GetBootstrapSecurityContextConstraints(buildControllerUsername string) []ka
 				// by the admission controller.  Admission will first look for the SupplementalGroupsAnnotation
 				// on the namespace and if it is unable to find that annotation it will attempt
 				// to use the UIDRangeAnnotation.  If neither annotation exists then creation
-				// of the SCC will fail.
+				// of the strategy will fail.
 				Type: kapi.SupplementalGroupsStrategyMustRunAs,
 			},
-			Groups: []string{AuthenticatedGroup},
 		},
+		// SecurityContextConstraintsAnyUID allows no host access and allocates SELinux.
```

Comment:
Ok, to make sure I'm clear on addressing this comment.  We have the following SCCs now:

1. privileged - all host access, priv access, RunAsAny for all strats
1. restricted - no host access, requires uid/selinux/fsgroup/sup grups
1. nonroot - same as restricted but with the NonRoot strategy for uid
1. hostmount - same as restricted but allows host mounts
1. hostaccess - same as restricted but allows access to all host ns features
1. anyuid - no host features, no priv, runAsAny for all strats except SELinux

Right now I've defaulted the authenticated group to anyuid and given a priority of 1 to privileged.  So all users have access to anyuid and, if you have access to privileged, you'll use it.

Is this incorrect?